docs/crio.conf.5.md

```diff
@@ -25,143 +23,188 @@ No bare options are used. The format of TOML can be simplified to:
+
+**container_exits_dir**="/var/run/crio/exits"
+  Path to directory in which container exit files are written to by conmon.
+
+**container_attach_socket_dir**="/var/run/crio"
+  Path to directory for container attach sockets.
+
+**read_only**=false
+  If set to true, all containers will run in read-only mode.
+
+**log_level**="error"
+  Changes the verbosity of the logs based on the level it is set to. Options are fatal, panic, error, warn, info, and debug.
+
+**uid_mappings**=""
+  The UID mappings for the user namespace of each container. A range is specified in the form containerUID:HostUID:Size. Multiple ranges must be separated by comma.
+
+**gid_mappings**=""
+  The GID mappings for the user namespace of each container. A range is specified in the form containerGID:HostGID:Size. Multiple ranges must be separated by comma.
+
+**ctr_stop_timeout**=10
+  The minimal amount of time in seconds to wait before issuing a timeout regarding the proper termination of the container.
 
-**read_only**==*true*|*false*
-  Run every container in read-only mode. Automatically mount tmpfs on `/run`, `/tmp` and `/var/tmp`.
-  Setup images to run in read-only. (default: false)
 
 ## CRIO.IMAGE TABLE
+The `crio.image` table contains settings pertaining to the management of OCI images.
 
-**default_transport**
-  A prefix to prepend to image names that can't be pulled as-is (default: "docker://")
+**default_transport**="docker://"
+  A prefix we for images that cannot be parsed as a valid source reference.
 
-**image_volumes**=""
-  Image volume handling ('mkdir', 'bind' or 'ignore') (default: "mkdir")
-  mkdir: A directory is created inside the container root filesystem for the volumes.
-  bind: A directory is created inside container state directory and bind mounted into
-  the container for the volumes.
-  ignore: All volumes are just ignored and no action is taken.
+**pause_image**="k8s.gcr.io/pause:3.1"
+  The image used to instantiate infra containers.
 
-**insecure_registries**=""
-  Enable insecure registry  communication,  i.e.,  enable  un-encrypted
-  and/or untrusted communication.
+**pause_command**="/pause"
+  The command to run in a `pause_image` to have a container just sit there. If the image contains the necessary information, this value does not need to be specified.
 
-  List  of  insecure registries can contain an element with CIDR notation
-  to specify a whole  subnet.  Insecure  registries  accept  HTTP  and/or
-  accept HTTPS with certificates from unknown CAs.
+**signature_policy**="/etc/containers/policy.json"
+  Path the file which decides what sort of policy we use when deciding whether or not to trust an image that we've pulled. Please refer to containers-policy.json(5) for more details.
```

Comment:
in our other docs we've added a warning to not change this.  For instance Buildah/Podman do:

Pathname  of  a  signature  policy  file to use.  It is not recommended that this option be used, as the
       
Pathname  of  a  signature  policy  file to use.  It is not recommended that this option be used, as the
default behavior of using the system-wide default  policy  (frequently  /etc/containers/policy.json)  is
most often preferred.

It would probably be good to layer such a warning here.